csrc/quantization/cutlass_w8a8/scaled_mm_c3x.cu

```diff
@@ -1,384 +1,22 @@
-// clang-format will break include orders
+// switch off clang format as the include statement indentation is inconsistent.
```

Comment:
Using clang-format in this block, turns, 
```
#include <cudaTypedefs.h>

#if defined CUDA_VERSION && CUDA_VERSION >= 12000

#include "scaled_mm_c3x_sm90_fp8_dispatch.cuh"
#include "scaled_mm_c3x_sm90_int8_dispatch.cuh"

#include "cutlass_extensions/epilogue/scaled_mm_epilogues_c3x.hpp"

using namespace vllm;
```

into 

```
#include <cudaTypedefs.h>

#if defined CUDA_VERSION && CUDA_VERSION >= 12000

    #include "scaled_mm_c3x_sm90_fp8_dispatch.cuh"
    #include "scaled_mm_c3x_sm90_int8_dispatch.cuh"

    #include "cutlass_extensions/epilogue/scaled_mm_epilogues_c3x.hpp"

using namespace vllm;
```
the `#if` seems trigger inconsistent indenting. I switched off clang-format in this block to avoid that. 

I moved the original clang-format toggle to `scaled_mm_c3x.cuh`. https://github.com/vllm-project/vllm/blob/dad8e4760f9d24faadfc076a13ed7d130211ea2d/csrc/quantization/cutlass_w8a8/scaled_mm_c3x.cuh#L3

Reply:
Had second thoughts about this and removed the `clang-format` block. It is probably better to stick to the convention.